tracker/src/main/kotlin/org/dhis2/tracker/relationships/ui/RelationshipsScreen.kt

```diff
@@ -0,0 +1,306 @@
+                modifier = Modifier
+                    .weight(1f)
+                    .fillMaxWidth()
+                    .align(Alignment.CenterVertically),
+            ) {
+                Title(text = title)
+
+                description?.let {
+                    Description(
+                        text = it,
+                        textColor = TextColor.OnSurfaceLight,
+                    )
+                }
+            }
+            if(canAddRelationship) {
+                IconButton(
+                    modifier = Modifier.testTag(TEST_ADD_RELATIONSHIP_BUTTON),
+                    style = IconButtonStyle.FILLED,
+                    icon = {
+                        Icon(
+                            imageVector = Icons.Outlined.Add,
+                            contentDescription = "New event",
+                            tint = MaterialTheme.colors.onPrimary,
+                        )
+                    },
+                    onClick = onCreateRelationshipClick,
+                )
+            }
+        }
+        relationships.forEach { item ->
+            ListCard(
+                listCardState = rememberListCardState(
+                    title = ListCardTitleModel(text = item.title),
+                    description = item.description?.let { ListCardDescriptionModel(text = it) },
+                    lastUpdated = item.lastUpdated,
+                    additionalInfoColumnState = rememberAdditionalInfoColumnState(
+                        additionalInfoList = item.attributes.map {
+                            AdditionalInfoItem(
+                                key = it.first,
+                                value = it.second,
+                            )
+                        },
+                        syncProgressItem = AdditionalInfoItem(
+                            key = stringResource(id = R.string.syncing),
+                            value = "",
+                        ),
+                        minItemsToShow = 3,
+                        expandLabelText = stringResource(id = R.string.show_more),
+                        shrinkLabelText = stringResource(id = R.string.show_less),
+                        scrollableContent = false,
+                    ),
+                ),
+                listAvatar = {
+                    AvatarProvider(
+                        avatarProviderConfiguration = item.avatar,
+                    ) { }
+                },
+                onCardClick = { if (item.canOpen) onRelationshipClick(item) }
+            )
+        }
+    }
+}
+
+@Composable
+private fun NoRelationships() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.White)
+            .padding(42.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center,
+    ) {
+        Image(
+            modifier = Modifier
+                .padding(1.dp)
+                .width(139.dp)
```

Comment:
maybe these hardcoded values for width and height won't work well across devices with different resolution specs?